js-exercises/update-object/updateObject.js

```diff
@@ -0,0 +1,17 @@
+
+function updateObject(indexParam, replacementCharacter, inputArray) {
+  let index = indexParam;
+  if (index >= inputArray.length) {
+    return -1;
+  }
+  if (index < 0) {
+    index = inputArray.length + index;
+  }
+  let outputArray = inputArray;
+  outputArray[index] = replacementCharacter;
+  return outputArray;
```

Comment:
This code fails when it receives an object. The problems says it is to update an object. So, please consider that as well.